content/docs/howto/dotnet-core.md

```diff
@@ -315,6 +315,84 @@ access the SBOM supplied by the buildpacks.
 
 SBOMs will be generated for all supported .NET Core applications.
 
+## Enable Remote Debugging
+
+Remote debugging can provide insight into complex program logic and
+interactions in remote environments. This practice is supported for .NET Core
+applications via the Visual Studio Debugger (`vsdbg`), which may be included in
+your application image via the BP_DEBUG_ENABLED environment variable. The
+debugger can attach to a running dotnet process and be bound to a client-side
```

Comment:
Just to clarify is this a process that is a .NET app or is this a process that is invoked by the `dotnet` cli?

Reply:
`dotnet` CLI, I'll make that more obvious.

Reply:
So I looked into this a bit and I realize I may have been incorrect above. The implication in [these docs](https://github.com/Microsoft/MIEngine/wiki/Offroad-Debugging-of-.NET-Core-on-Linux---OSX-from-Visual-Studio#linux-computer) seemed to be that the `dotnet` CLI is a requirement for this functionality. The excerpt below, taken from [this section](https://github.com/Microsoft/MIEngine/wiki/Offroad-Debugging-of-.NET-Core-on-Linux---OSX-from-Visual-Studio#create-launch-configuration-file) also seems to suggest that:

> Program: The path, on the Linux computer, to the executable I want to run. Since the debuggee is a .NET Core assembly, vsdbg will automatically run the program with the dotnet host executable.

All that being said, I tried this with one of our "self-contained" testing apps and nothing seemed out of place when attempting to run the debugger. (It also didn't give much useful information, but it did not fail as far as I can tell).

Reply:
[This](https://github.com/OmniSharp/omnisharp-vscode/wiki/Attaching-to-remote-processes#building-and-deploying-the-application-and-pdbs) should clear things up on the self-contained front.

> PDB files: In order for VSDBG to be able to be able to map executable code back to its source code (or vice versa) VSDBG needs to have PDB files.

Self-contained app images would have PDB files present, so all should be good.

Reply:
This is still not super clear to me through the docs that you have written here.